package.json

```diff
@@ -22,5 +24,11 @@
   "bugs": {
     "url": "https://github.com/NoroffFEU/css-frameworks-ca/issues"
   },
-  "homepage": "https://github.com/NoroffFEU/css-frameworks-ca#readme"
+  "homepage": "https://github.com/NoroffFEU/css-frameworks-ca#readme",
+  "devDependencies": {
+    "bootstrap": "^5.2.0",
+    "live-server": "^1.2.2",
+    "node-sass": "^6.0.1",
```

Comment:
You can uninstall node-sass:

`npm uninstall node-sass`

Reply:
started all over again: node-sass is not installed this time and is not to befound under devDependencies